### ISSUE TITLE
Song menu 

### ISSUE THREAD

**Issue by ddacw** [HaikuArchives/GuitarMaster#2]
* contains files that are not songs.
* contains folders but can't open them.
* does not indicate which file can be opened.

**Comment by scottmc** [comment]
seems to be stuck looking only at home folder?

**Comment by BachToTheFuture** [comment]
GuitarMaster is suppose to read MIDI files.
However, I tested this app with several MIDI files and it can't read any of them.
TIP: If you run GuitarMaster as a command from Terminal you should be able to see what it's doing.
I also found this:
```
	BString * songLocations[13];
	songLocations[0] = UserSpecifiedSongDir;
	songLocations[1] = new BString("songs");
	songLocations[2] = new BString("/music/GuitarMaster");
	songLocations[2]->Prepend(homeDir);
	songLocations[3] = new BString("/music/guitarmaster");
	songLocations[3]->Prepend(homeDir);
	songLocations[4] = new BString("/music/Guitar Master");
	songLocations[4]->Prepend(homeDir);
	songLocations[5] = new BString("/music/guitar master");
	songLocations[5]->Prepend(homeDir);
	songLocations[6] = new BString("/Music/GuitarMaster");
	songLocations[6]->Prepend(homeDir);
	songLocations[7] = new BString("/Music/guitarmaster");
	songLocations[7]->Prepend(homeDir);
	songLocations[8] = new BString("/Music/Guitar Master");
	songLocations[8]->Prepend(homeDir);
	songLocations[9] = new BString("/Music/guitar master");
	songLocations[9]->Prepend(homeDir);
	songLocations[10] = new BString("/music");
	songLocations[10]->Prepend(homeDir);
	songLocations[11] = new BString("/Music");
	songLocations[11]->Prepend(homeDir);
	songLocations[12] = new BString(homeDir);
```
Seems like GuitarMaster _used_ to have its custom songs, but somehow it's not included in this repo ... which is obviously a problem 🙁 
Since it can't find any of those locations, it resorts to `homeDir` instead.

**Comment by ddacw** [comment]
@bach5000
Indeed. I even tried some 'Frets on Fire' song. No luck, obviously.
In the end they're just green ~~rows~~ columns. 

_Maybe this app could be used to make a PvZ clone._

**Comment by eddrda** [comment]
*columns, actually. Maybe this app could be used to make a PvZ clone.

**Comment by richienyhus** [comment]
Maybe @Lt-Henry could tell us where those custom songs are?

**Comment by Lt-Henry** [comment]
hmfs! This application is an unfinished attempt to clone frets on fire on Haiku, but it is far far far from being usable. I have no idea how it ended here. it was planned to work with frets on fire midi files, but I need to take a look at code (I wrote it almost 9 years ago)

**Comment by pulkomandy** [comment]
The HaikuArchives team may be a bit overzealous sometimes. Wouldn't it be simpler to port the actual Frets on Fire?

**Comment by Lt-Henry** [comment]
I'm a bit outdated with current Haiku status, the actual Frets on Fire package on ubuntu has following dependencies:  fonts-mgopen, python, python-imaging, python-ogg, python-opengl, python-pygame, python-pyvorbis, fonts-dejavu

Imho, lacking hardware acceleration, i will try to render with some simple software blits instead of software OpenGL. My guess is that Frets on Fire will be unplayable without a hardware OpenGL.
